.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: go
 
 go:
-  - 1.4
-  - 1.5
-  - 1.6
-  - tip
+ - 1.5.x
+ - 1.6.x
+ - 1.7.x
+ - 1.8.x
+ - 1.9.x
+ - master
```

Comment:
Nitpick: YML files most commonly use 2 spaces for indentation, not one. Is there a specific reason you changed it from 2 to 1?

Reply:
Fixed it and you were right to point it out. I was using then GitHub editor out of convenience which was set to 1 space for some reason and I didn't noticed in time.